### PR TITLE
Attachments for assessor

### PR DESCRIPTION
**Pull request by jaleszek**
Exposed attachment resource for the Assessor. Extracted the module with shared methods to reuse it in different controller namespaces. Probably something to discuss as not hyper popular in Rails world but looks as easy way to achieving non dry controllers for Assessor/Admin spaces.

### CODE REVIEW

File: app/controllers/concerns/form_answer_attachments_context.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
+module FormAnswerAttachmentsContext
+  def create
```
Comment (matthewford) [inline app/controllers/concerns/form_answer_attachments_context.rb]: Assignment Branch Condition size for create is too high. [15.17/15]

**Comment by jaleszek** [comment]
Does anyone can review this pr please ?

**Comment by RuslanHamidullin** [comment]
yeP )

**Comment by RuslanHamidullin** [comment]
Good to merge :+1:

**Pull request merged**